src/lv_core/lv_indev.h

```diff
@@ -141,6 +141,13 @@ void lv_indev_wait_release(lv_indev_t * indev);
  */
 lv_task_t * lv_indev_get_read_task(lv_disp_t * indev);
 
+/**
+ * Gets a pointer to the currently handled object.
+ * NULL if no object is currently being handled or if groups aren't used.
+ * @return pointer to currently focused object
+ */
+lv_obj_t * lv_indev_get_obj_focused();
```

Comment:
If we use this function to get the currently handled object than it can be used for `POINTER` types too. So a name like `lv_indev_get_obj_act` would be more descriptive. What do you think? If you agree I can add the `POINTER` related part.

Reply:
Sounds good, I'll change the name so you can add the `POINTER` code